reladomo/src/test-util/java/com/gs/fw/common/mithra/test/MithraTestResource.java

```diff
@@ -155,6 +155,7 @@ protected MithraTestResource(String configFilename, DatabaseType databaseType, M
         getLogger().debug("config file " + configFilename);
 
         this.runtimeType = this.loadConfigXml(configFilename);
+        this.initializeRuntimeConfig(runtimeType);

```

Comment:
this is the wrong time to initialize the runtime. The test resource has a lifecycle that looks like this:
1) Construct the object. This can happen at or before junit `setup`.
2) During junit `setup` (aka `@Before`), call `resource.setup()`. This is when the runtime should be initialized.
3) During junit `tearDown` (aka `@After`). call `resource.tearDown()`

Some advanced uses of the test resource can optimize by setting up the resource once per test class.